src/scene/container/RenderContainer.ts

```diff
@@ -106,9 +77,6 @@ export class RenderContainer extends Container implements View, Instruction
         });
 
         if (render) this.render = render;
-
-        this.containsPoint = options.containsPoint ?? (() => false);
-        this.addBounds = options.addBounds ?? (() => false);
```

Comment:
why have these been removed?

Reply:
we can rely on the default functions in ViewContainer instead

Reply:
but this removes the ability to set these functions through the constructor, so would be a breaking change?

Reply:
true! have added in some backwrds compatability!